tests/QAN/pages/qanOverviewFragment.js

```diff
@@ -221,4 +225,12 @@ module.exports = {
     I.fillField(this.fields.searchBy, value);
     I.pressKey('Enter');
   },
+
+  getToTimeFromUrl(url) {
+    return url.slice(url.indexOf('&to=') + 4, url.indexOf('&page_number='));
```

Comment:
please use [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL_API#queries) to parse URLs. This way is not reliable

Reply:
ok. thank you.
edited